packages/xr/svelte.config.js

```diff
@@ -1,5 +1,5 @@
 import adapter from '@sveltejs/adapter-auto'
-import { vitePreprocess } from '@sveltejs/kit/vite'
+import { vitePreprocess } from '@sveltejs/vite-plugin-svelte'
```

Comment:
This import needed to be updated. The other packages use `svelte-preprocess`, a custom preprocessor and don't need to be updated.

Side discussion: should we remove `svelte-preprocess` and replace with this import? I could see this potentially causing issues for users if someone accidentally contributes a change that works with `svelte-preprocess` but a user is using the default `vitePreprocess`.

Reply:
There are two points where `<style lang="postcss">` are used. 1. `RightSidebar.svelte` and 2. a bunch of flex `DOM.svelte` files.

1. Would require inlining the tailwind classes
2. would just be an explicit copy paste where needed and would remove a bunch of warnings in the build output 👍 

+1 to inlining from me but maybe there was some reason they weren't inlined that I'm missing

Reply:
Oh I should have specified: I think it would make sense only to remove this plugin for packages that are shipped to NPM, since we ship the svelte files. For the docs I think it totally makes sense to keep this plugin + tailwind for DX.